example/src/main/java/org/apache/rocketmq/example/schedule/ScheduledMessageConsumer.java

```diff
@@ -17,31 +17,33 @@
 package org.apache.rocketmq.example.schedule;
 
 import org.apache.rocketmq.client.consumer.DefaultMQPushConsumer;
-import org.apache.rocketmq.client.consumer.listener.ConsumeConcurrentlyContext;
 import org.apache.rocketmq.client.consumer.listener.ConsumeConcurrentlyStatus;
 import org.apache.rocketmq.client.consumer.listener.MessageListenerConcurrently;
 import org.apache.rocketmq.common.message.MessageExt;
 
-import java.util.List;
-
 public class ScheduledMessageConsumer {
-    
+
+    public static final String CONSUMER_GROUP = "ExampleConsumer";
+    public static final String DEFAULT_NAMESRVADDR = "127.0.0.1:9876";
+    public static final String TOPIC = "TestTopic";
+
     public static void main(String[] args) throws Exception {
         // Instantiate message consumer
-        DefaultMQPushConsumer consumer = new DefaultMQPushConsumer("ExampleConsumer");
+        DefaultMQPushConsumer consumer = new DefaultMQPushConsumer(CONSUMER_GROUP);
+
+        // If the debugging source code can open comments, you need to set the namesrvAddr to the address of the local namesrvAddr
+//        consumer.setNamesrvAddr(DEFAULT_NAMESRVADDR);
+
         // Subscribe topics
-        consumer.subscribe("TestTopic", "*");
+        consumer.subscribe(TOPIC, "*");
         // Register message listener
-        consumer.registerMessageListener(new MessageListenerConcurrently() {
-            @Override
-            public ConsumeConcurrentlyStatus consumeMessage(List<MessageExt> messages, ConsumeConcurrentlyContext context) {
-                for (MessageExt message : messages) {
-                    // Print approximate delay time period
-                    System.out.printf("Receive message[msgId=%s %d  ms later]\n", message.getMsgId(),
-                            System.currentTimeMillis() - message.getStoreTimestamp());
-                }
-                return ConsumeConcurrentlyStatus.CONSUME_SUCCESS;
+        consumer.registerMessageListener((MessageListenerConcurrently) (messages, context) -> {
```

Comment:
Think this code is suppose to compile under jdk 1.7, so lambda may not supported

Reply:
> Think this code is suppose to compile under jdk 1.7, so lambda may not supported

<img width="661" alt="image" src="https://user-images.githubusercontent.com/34903552/169732580-639a1b17-2aa0-47ac-8fa6-216ab618f644.png">
Rocketmq is compiled under 1.8 thank you